src/Module/Conversation/Community.php

```diff
@@ -356,7 +356,7 @@ private static function selectItems($min_id, $max_id, $item_id, $itemspage)
 			}
 		}
 
-		$r = Post::selectThreadForUser(0, ['uri-id', 'commented', 'author-link'], $condition, $params);
+		$r = Post::selectThreadForUser(DI::userSession()->getLocalUserId() ?: 0, ['uri-id', 'commented', 'author-link'], $condition, $params);
```

Comment:
@annando This change allows for remote server local user settings to be taken into account in the Global Community page. Can you see any drawbacks of doing this?

Reply:
One thing that I see here is that this only can cover posts that are started by a blocked server, excluding replies from contacts on blocked servers.

Reply:
How would I go about that?

Reply:
See here: https://github.com/friendica/friendica/blob/develop/src/Content/Conversation.php#L955-L992C4

Reply:
Thanks, the behavior now matches the description.